testsuite/cluster-test/src/experiments/cpu_flamegraph.rs

```diff
@@ -66,7 +66,7 @@ impl Experiment for CpuFlamegraph {
             .tx_emitter
             .emit_txn_for(tx_emitter_duration, emit_job_request)
             .boxed();
-        let run_id = env::var("RUN_ID").expect("RUN_ID is not set");
```

Comment:
what does this obtain? what happens if we error here instead of expect?

Reply:
We run the other experiments in the `Vec<Box<dyn Experiment>>` list rather than stopping *everything* at this one.

Reply:
yeah, so this seems bad, since if the run_id isn't set here, why would it be set elsewhere? cc: @andll / @snakech 

I'd prefer these folks to weigh in...